client/src/components/History/model/queries.js

```diff
@@ -65,6 +65,39 @@ export async function updateContentFields(content, newFields = {}) {
     return doResponse(response);
 }
 
+/** Get all objects that match filters in a history
+ *
+ * @param {string} historyId
+ * @param {Object} filters
+ * @param {string} type
+ *
+ */
+export async function getHistoryContent(historyId, filters = {}, type = "dataset") {
+    const vDefault = { visible: true };
+    const dDefault = { deleted: false };
+    const pDefault = { purged: false };
+    //check if filters inclues hidden, deleted, or purged
+    const filterKeys = Object.keys(filters);
+    const hasVisible = filterKeys.includes("visible");
+    const hasDeleted = filterKeys.includes("deleted");
+    const hasPurged = filterKeys.includes("purged");
+    //if filters does not include hidden, deleted, or purged, set defaults to false
+    if (!hasVisible) {
+        filters = Object.assign({}, vDefault, filters);
+    }
+    if (!hasDeleted) {
+        filters = Object.assign({}, dDefault, filters);
+    }
+    if (!hasPurged) {
+        filters = Object.assign({}, pDefault, filters);
+    }
+
```

Comment:
I'm confused by these conditions, if those are the default values, you don't need to include them in the query right?

Your `filters` dictionary built with `HistoryFilters.getQueryDict(this.filterText)` should have all that you need, I would expect those default values to be redundant in the query unless I'm missing something :thinking:

Reply:
they're not default values in the actual request, but I feel they are default values for what the user would actually want. 

If I wanted to create a collection from everything in my history, I would expect it to only include the items I could see.